src/components/menu/menu.stories.mdx

```diff
@@ -183,6 +183,45 @@ is rendered.
   </Story>
 </Preview>
 
+## Default icon
```

Comment:
Is the much value in having both/ either icon only story captured by `chromatic`